### DISCUSSION TITLE
multivariate time series forecasting  is stuck

### DISCUSSION THREAD

**Discussion by ramankhurana** [ourownstory/neural_prophet#1111] [Q&A - get help using NeuralProphet]
Hello, 

I am trying to use NeuralProphet for multivariate time series forecasting. I found an example [1] in the documentation but I am unable run this example and I wonder if it is a known issue. . 

When i run model.fit the notebook keep throwing "Warning: NP.time_net.denormalize" forever. I attach the screenshot here. 
I am not doing anything extra, just copy pasting the lines from link [1] to a kaggle notebook.  

I wonder if anyone else face this. 
<img width="767" alt="image" src="https://user-images.githubusercontent.com/4996609/211957856-c8273483-7c22-44fe-88e0-60108c25169a.png">

[1]
https://neuralprophet.com/notebooks/global_modeling.html#Global-Model

**Comment by judussoari** [comment]
By updating to the latest NP version, this should not be a problem anymore. 
Let us know if this is still an Issue @ramankhurana :)
